cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -27,6 +27,12 @@
   end
 end
 
+When(/^I( (?:can|successfully))? GET "([^"]*)" with user "([^"]*)"$/) do |can, path, username|
+  try_request can do
+    get_json path, user: username
+  end
+end
+
```

Comment:
@jtuttle I added this step so we can run a GET request with the last retrieved token. Should I add also methods for PUT, POST & DELETE? I don't need them at the moment